manifests/bucketeer/charts/experiment-calculator/templates/envoy-configmap.yaml

```diff
@@ -0,0 +1,298 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "experiment-calculator.fullname" . }}-envoy-config
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ template "experiment-calculator.name" . }}
+    chart: {{ template "experiment-calculator.chart" . }}
+    release: {{ template "experiment-calculator.fullname" . }}
+    heritage: {{ .Release.Service }}
+data:
+  config.yaml: |-
+    admin:
+      access_log:
+        name: envoy.access_loggers.file
+        typed_config:
+          '@type': type.googleapis.com/envoy.extensions.access_loggers.file.v3.FileAccessLog
+          path: /dev/stdout
+      address:
+        socket_address:
+          address: 0.0.0.0
+          port_value: 8001
+    static_resources:
+      clusters:
+        - name: calculator
+          type: strict_dns
+          connect_timeout: 5s
+          dns_lookup_family: V4_ONLY
+          lb_policy: round_robin
+          load_assignment:
+            cluster_name: calculator
+            endpoints:
+              - lb_endpoints:
+                  - endpoint:
+                      address:
+                        socket_address:
+                          address: localhost
+                          port_value: 9090
+          transport_socket:
+            name: envoy.transport_sockets.tls
+            typed_config:
+              '@type': type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
+              common_tls_context:
+                alpn_protocols:
+                  - h2
+                tls_certificates:
+                  - certificate_chain:
+                      filename: /usr/local/certs/service/tls.crt
+                    private_key:
+                      filename: /usr/local/certs/service/tls.key
+          typed_extension_protocol_options:
+            envoy.extensions.upstreams.http.v3.HttpProtocolOptions:
+              '@type': type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
+              explicit_http_config:
+                http2_protocol_options: {}
+          health_checks:
+            - grpc_health_check: {}
+              healthy_threshold: 1
+              interval: 10s
+              interval_jitter: 1s
+              no_traffic_interval: 2s
+              timeout: 1s
+              unhealthy_threshold: 2
+          ignore_health_on_host_removal: true
+        - connect_timeout: 5s
+          dns_lookup_family: V4_ONLY
+          ignore_health_on_host_removal: true
+          lb_policy: round_robin
+          load_assignment:
+            cluster_name: environment
+            endpoints:
+              - lb_endpoints:
+                  - endpoint:
+                      address:
+                        socket_address:
+                          address: environment.{{ .Values.namespace }}.svc.cluster.local
+                          port_value: 9000
+          name: environment
```

Comment:
nit:
Please move the `name` field to the top(L:65) for consistency and visibility.